files/galaxy/dynamic_job_rules/production/total_perspective_vortex/destinations.yml.j2

```diff
@@ -347,6 +347,10 @@ destinations:
             docker_set_user: '1000'
             docker_memory: '{mem}G'
             docker_sudo: false
+      - id: pulsar_destination_docker_af_db_test_rule
+        params:
+            docker_volumes: $job_directory:ro,$job_directory/home:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/data2/alphafold:/data:ro
+        if: entity.context.get('alphafold_db_test')
```

Comment:
Hey @cat-bro will this work?

Reply:
https://github.com/usegalaxy-au/infrastructure/pull/2076/commits/3a2349df55df651218b005884b0e72da8dd1d7f5#diff-15a1354b6f63edcb97e94f03d4bcdacc8cc56295ddb062f416298837a0992026R3206-R3208

Reply:
https://github.com/usegalaxy-au/infrastructure/blob/3a2349df55df651218b005884b0e72da8dd1d7f5/files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml#L3206-L3208

Reply:
It's worth a try. Only alphafold test and dorado jobs going at the moment.

Reply:
Ok, fire away if you're happy